src/app/_models/question_report.ts

```diff
@@ -0,0 +1,6 @@
+export interface QuestionReport {
+    id: number;
+    question_id: number;
+    report: string;
```

Comment:
you should probably make this an enum since it can only have a few specified values

Reply:
We made report an enum!